tests/queries/test_explain.py

```diff
@@ -30,6 +32,12 @@ def test_basic(self):
                         result = queryset.explain(format=format)
                         self.assertTrue(captured_queries[0]['sql'].startswith(connection.ops.explain_prefix))
                         self.assertIsInstance(result, str)
+                        if format == 'json':
+                            # Parse the result to check it is valid JSON.
+                            json.loads(result)
+                        elif format == 'xml':
+                            # Parse the result to check it is valid XML.
+                            xml.etree.ElementTree.fromstring(result)
```

Comment:
Please move this to a separate commit as an extra test coverage.

Reply:
Done